tutorials/getting_started/tips-and-tricks.md

```diff
@@ -62,6 +62,8 @@ The developer sidebar search ignores case and searches inside the following enti
   - Item name & Thing UID of triggers, actions & conditions
   - script code (e.g. Rules DSL, JavaScript, and even Blockly)
   - script MIME types (requires exact match)
+  - lists Blockly scripts (using 'block', 'blockly' or 'blocksource' as search string)
```

Comment:
```suggestion
  - Blockly scripts (using 'block', 'blockly' or 'blocksource' as search string, where case is ignored)
```

IMO the removal if `lists` aligns it with the other points. I also think it should be mentioned that case is ignored for the search string.

Reply:
Updated

Reply:
looks good to me.